.gitlab-ci.yml

```diff
@@ -44,6 +48,22 @@ build:backend:dist:
     paths:
       - dist.tar.gz
 
+build:docker:
+  image: golang:${GOLANG_VERSION}
+  stage: build
+  needs:
+    - job: build:binaries
+      artifacts: true
+  rules:
+    - changes:
+        paths: ["backend/**/*"]
+        compare_to: "${RULES_CHANGES_COMPARE_TO_REF}"
+  tags:
+    - hetzner-amd-beefy
+  script:
+    - make docker
```

Comment:
do we really want to rebuild all the services for a single service change? This will unnecessarily slow down the pipeline

Reply:
Additionally, it could be wonderful if we can use parallel jobs when we have to build multiple changed services.